support-workers/src/main/scala/com/gu/config/Configuration.scala

```diff
@@ -4,11 +4,14 @@ import com.gu.config.loaders.PrivateConfigLoader
 import com.gu.monitoring.SafeLogger
 import com.gu.salesforce.SalesforceConfigProvider
 import com.gu.support.config._
-import com.typesafe.config.ConfigFactory
+import com.typesafe.config.{Config, ConfigFactory}
 
 import scala.util.Try
 
 object Configuration {
+
+  def load(): Configuration = new Configuration(loadConfig)
```

Comment:
it only actually loads the first time, but I wasn't sure of the best name.  It should only be called once "at the edge" anyway.